files/packit.wsgi

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+from packit.service.web_hook import app as application
```

Comment:
flake8 has to be disabled for `files/packit.wsgi`. This is usual calling Flask application from WSGI.
See:
http://flask.pocoo.org/docs/1.0/deploying/mod_wsgi/
https://www.jakowicz.com/flask-apache-wsgi/
https://www.bogotobogo.com/python/Flask/Python_Flask_HelloWorld_App_with_Apache_WSGI_Ubuntu14.php

Reply:
I don't understand. Why it has to be disabled? I just tried `pre-commit run --all-files` in your branch and everything passed.

Reply:
It failed for this reason:

~~~
Check for added large files..............................................Passed
Check python ast.........................................................Passed
Check for merge conflicts................................................Passed
Check Yaml...............................................................Passed
Detect Private Key.......................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
Flake8...................................................................Failed
hookid: flake8

files/packit.wsgi:3:1: F401 'packit.service.web_hook.app as application' imported but unused
~~~

Reply:
Ok, we can [ignore it](http://flake8.pycqa.org/en/latest/user/options.html?highlight=per-file-ignores#cmdoption-flake8-per-file-ignores) then.
Try to add `--per-file-ignores='files/packit.wsgi:F401'` to [flake8 args](https://github.com/packit-service/packit/blob/master/.pre-commit-config.yaml#L17)

Reply:
Aha, I didn't see you had already excluded the `files/`. That's why it passed for me.